docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -418,18 +409,27 @@ First, it is determined if the simulant will have a healthcare interaction in th
     - Decision Information 
     - Notes
   * - A
-    - ASCVD = -19.5 + (0.043 * SBP) + (0.266 * Age) + (2.32 * Sex)
-    -  
+    - ASCVD = -19.5 + (0.043 * SBP) + (0.266 * Age) + (2.32 * Sex) where Sex=1 for males and Sex=0 for females 
+    - 
   * - B
-    - LDL-C measreument error pulled from a normal distribution with mean=0 and SD=3 mg/dL    
+    - LDL-C measreument error pulled from a normal distribution with mean=0 and SD=0.08 mmol/L    
     - [McCormack_2020]_
   * - C
-    - 19.4% will not start medication; 14.2% will receive high intensity statin; 55.1% medium intensity; and 11.3% low intensity 
-    - [Morales_2018]_ [Arnett_2019]_
+    - If simulant is in the acute or post MI or stroke states  
+    - [Morales_2018]_ [Arnett_2019]_ [Nguyen_2015]_
```

Comment:
Is there an error here? I Don't see instructions here just "if simulant is in...". Just confirming!

Reply:
This is probably not a needed line in the table